### PR TITLE
[fix test issues] extend time to do unattended-upgrade

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: microsoft/testsuites/core/dns.py [hunk 1]
```diff
@@ -48,13 +48,28 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
             )
             if 0 != cmd_result.exit_code:
                 node.os.install_packages("unattended-upgrades")
+            if not isinstance(node.os, Ubuntu):
```
Comment (squirrelsc) [inline microsoft/testsuites/core/dns.py]: Does it cover Debian only? If so, use `type(node.os) == Debian)`

File: microsoft/testsuites/core/dns.py [hunk 2]
```diff
@@ -48,13 +48,28 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
...
+                if 10 == node.os.information.version.major:
```
Comment (squirrelsc) [inline microsoft/testsuites/core/dns.py]: 1. Use version compare pattern like `node.os.information.version >= 10.0.0`.
2. As above pattern, always assume higher version use the same mechanism, until we found it's wrong.
3. Always use an `else` to handle rest versions.

File: microsoft/testsuites/core/dns.py [hunk 3]
```diff
@@ -48,13 +48,28 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
...
+                    node.execute(
+                        "mkdir -p /var/cache/apt/archives/partial",
+                        sudo=True,
+                        shell=True,
+                        expected_exit_code=0,
+                        expected_exit_code_failure_message=(
+                            "fail to make folder /var/cache/apt/archives/partial"
+                        ),
+                    )
+                if 9 == node.os.information.version.major:
```
Comment (squirrelsc) [inline microsoft/testsuites/core/dns.py]: `elif`

File: microsoft/testsuites/core/dns.py (hunk 1 above)
Comment (LiliDeng) [reply microsoft/testsuites/core/dns.py]: yes, updated.

File: microsoft/testsuites/core/dns.py (hunk 2 above)
Comment (LiliDeng) [reply microsoft/testsuites/core/dns.py]: updated

File: microsoft/testsuites/core/dns.py (hunk 3 above)
Comment (LiliDeng) [reply microsoft/testsuites/core/dns.py]: updated

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**